### ISSUE TITLE
Missing Administration, Monitoring, and Deployment guides

### ISSUE THREAD

**Issue by jonny-novikov** [EnterpriseQualityCoding/FizzBuzzEnterpriseEdition#325]
Our IT department has a lack of documentation to deploy solution to our enterprise cluster. Can you provide some?

**Comment by vans163** [comment]
Yes we can please upgrade to the Enterprise for the Enterprise, Enterprise license, you currently have a Enterprise for Enterprise Plus.  This license you current have does not cover enterprise clustering, nor will you receive any suport.

**Comment by jonny-novikov** [comment]
Any updates on this issue?
